apps/src/lab2/views/components/TextToSpeech.module.scss

```diff
@@ -3,13 +3,27 @@
 
 .playButton {
   @include remove-button-styles;
+  display: flex;
   position: absolute;
-  top: 6px;
-  right: 6px;
-  opacity: 50%;
+  bottom: 7px;
+  right: 7px;
```

Comment:
It's also possible to use `inset-inline-end: 7px` here (it automatically perses left/right accordingly for ltr/rtl languages ), this we won't need to add  
 ```
html[dir="rtl"] & {
    right: initial;
    left: 6px;
  }
```

Reply:
Thanks for the reminder; updated.